src-ghc/Pact/Server/PactService.hs

```diff
@@ -63,6 +61,10 @@ applyCmd logger conf dbv cv exMode _ (ProcSucc cmd) = do
   case r of
     Right cr -> do
       logLog logger "DEBUG" $ "success for requestKey: " ++ show (cmdToRequestKey cmd)
+      -- FOR TESTING
+      s <- liftIO $ readMVar cv
+      let pactState = (_csPacts s)
+      logLog logger "DEBUG" $ "testing that commandstate saved: " ++ show pactState
```

Comment:
figure out how to make this a meaningful DEBUG log as I'm sure we'll be interested in seeing pact state saved when debugging pact scripts, ie maybe show the one that got saved instead of the whole map

Reply:
See "Added better debug logging" commit